prow/daily-e2e-cluster_wide-auth.sh

```diff
@@ -33,7 +33,8 @@ fi
 
 # Exports $HUB, $TAG, and $ISTIOCTL_URL
 source greenBuild.VERSION
-echo "Using artifacts from HUB=${HUB} TAG=${TAG} ISTIOCTL_URL=${ISTIOCTL_URL}"
+ISTIOCTL_STAGE_URL=${ISTIOCTL_URL}-stage
+echo "Using artifacts from HUB=${HUB} TAG=${TAG} ISTIOCTL_STAGE_URL=${ISTIOCTL_STAGE_URL}"
```

Comment:
echo ISTIOCTL_STAGE_URL here

Reply:
Hmm, I am not quiet sure what you mean. I think I do echo the `ISTIOCTL_STAGE_URL ` here :)

Reply:
Can we just update ${ISTIOCTL_URL} to ${ISTIOCTL_URL}-stage so it will just be a one line change after istio/istio#2904 (comment) is addressed?

Reply:
We are using `ISTIOCTL_URL` somewhere else in this script, so it would be good to keep it.